src/autowiring/test/AutowiringDebugTest.cpp

```diff
@@ -56,13 +57,13 @@ TEST_F(AutowiringDebugTest, ContextPrintout) {
 
   // This is the expected output
   std::string tree = "void(Current Context)\n"
-                     "├── Derp\n"
-                     "│   ├── Herp\n"
-                     "│   │   └── int\n"
-                     "│   └── Derp\n"
-                     "└── Herp\n"
-                     "    ├── Derp\n"
-                     "    └── Derp\n";
+                     "|-- Derp\n"
+                     "|   |-- Herp\n"
+                     "|   |   *-- int\n"
+                     "|   *-- Derp\n"
+                     "*-- Herp\n"
+                     "    |-- Derp\n"
+                     "    *-- Derp\n";
```

Comment:
The unicode characters don't work properly on Windows consoles.  Had to swap it out for this stuff instead.